backend/src/Infrastructure/Data/Config/ClientConfiguration.cs

```diff
@@ -9,7 +9,7 @@ internal class ClientConfiguration : IEntityTypeConfiguration<Client>
         public void Configure(EntityTypeBuilder<Client> builder)
         {
             builder
-                .Property(x => x.ClientId)
+                .Property(x => x.Id)
```

Comment:
To powinno zostać jako ClientId, poza tym jeśli miało by to być zmienione na long to HasMaxLength jest zupełnie bez sensu.